app/static/openapi.yaml

```diff
@@ -943,11 +947,10 @@ components:
             {
               'apiVersion': '1.0',
               'data': null,
-              'errors': [
-                  {
-                      'code': 'not-found'
-                  }
-              ],
+              'errors': {
+                'not-found': {
+                  'message': "The resource you requested does not exist.'
```

Comment:
This should be `'The resource...` instead of `"The resource...` because it renders with a `\"` when you use the double quotes